docs/list.js

```diff
@@ -818,14 +821,15 @@ var list = {
 				"Lensflare": "examples/zh/objects/Lensflare",
 			},
 
-			"Post-Processing": {
+			"后期处理": {
 				"EffectComposer": "examples/zh/postprocessing/EffectComposer"
 			},
 
 			"导出器": {
 				"GLTFExporter": "examples/zh/exporters/GLTFExporter",
-				"PLYExporter": "examples/zh/exporters/PLYExporter"
-			},
+				"PLYExporter": "examples/zh/exporters/PLYExporter",
+                "ColladaExporter": "examples/zh/exporters/ColladaExporter"
```

Comment:
Some minor formatting issues.

BTW: When updating the docs in the past, I've noticed that some zh pages mix hard and soft tabs. Would be good to just use hard tabs.b

Reply:
Thanks for pointing, I will pay attention to the issue later ,and try my best to fix it.